library/foundations/scripts/lib/token-parser.ts

```diff
@@ -563,7 +563,7 @@ export const processLayers = <T extends TokenStructure>(
         isStatic,
         parameters[PARAM_SECTION_NAME]
       );
-      return { name, variables, parameters, isDynamic: isStatic };
+      return { name, variables, parameters, isStatic };
     })
```

Comment:
this was a mistake. I renamed the variable but it didn’t rename the way it was being exported.
There’s a pending change: ensure the type of the output (it would have prevented this issue)